From 99361f28c8763ea4569b5ae9876028d53ac9a234 Mon Sep 17 00:00:00 2001
From: Alberts MuktupƒÅvels <alberts.muktupavels@gmail.com>
Date: Sun, 24 May 2020 13:58:51 +0300
Subject: [PATCH] compositor-vulkan: fix build

VK_PHYSICAL_DEVICE_TYPE_RANGE_SIZE enum was removed in 1.2.140.

https://gitlab.gnome.org/GNOME/metacity/-/issues/10
---
 src/compositor/meta-compositor-vulkan.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git src/compositor/meta-compositor-vulkan.c src/compositor/meta-compositor-vulkan.c
index b87b5194..53b7b3ff 100644
--- src/compositor/meta-compositor-vulkan.c
+++ src/compositor/meta-compositor-vulkan.c
@@ -810,7 +810,9 @@ device_type_to_string (VkPhysicalDeviceType type)
         return "cpu";
         break;
 
+#if VK_HEADER_VERSION < 140
       case VK_PHYSICAL_DEVICE_TYPE_RANGE_SIZE:
+#endif
       case VK_PHYSICAL_DEVICE_TYPE_MAX_ENUM:
       default:
         break;
-- 
GitLab

